src/ophys_etl/transforms/convert_rois.py

```diff
@@ -0,0 +1,193 @@
+    )
+
+    @mm.post_load()
+    def check_args(self, data, **kwargs):
+        if data['binary_quantile'] is not None:
+            if data['binary_quantile'] <= 0:
+                raise BinarizeAndCreationException("Binary quantile must be"
+                                                   "positive value")
+        if data['abs_threshold'] is not None:
+            if data['abs_threshold'] <= 0:
+                raise BinarizeAndCreationException("Abs threshold must be "
+                                                   "positive value")
+        return data
+
+    @mm.post_load()
+    def check_file_existance(self, data, **kwargs):
+        """
+        have to check manually for existance of .h5 and .npy files because
+        InputFile fails on these, at least on windows because of a use of
+        generic open() to check for existance.
+        """
+        if not Path(data['suite2p_stat_path']).exists():
+            raise BinarizeAndCreationException("suite2p_stat_path supplied "
+                                               "is not a valid path.")
+        if not Path(data['motion_corrected_video']).exists():
+            raise BinarizeAndCreationException("motion_corrected_video "
+                                               "supplied is not a valid path.")
+        return data
+
+
+class OldSegmentationROISchema(ArgSchema):
+    id = Int(required=True,
+             description=("Unique ID of the ROI, get's overwritten writting "
+                          "to LIMS"))
+    x = Int(required=True,
+            description="X location of top left corner of ROI in pixels")
+    y = Int(required=True,
+            description="Y location of top left corner of ROI in pixels")
+    width = Int(required=True,
+                description="Width of the ROI in pixels")
+    height = Int(required=True,
+                 description="Height of the ROI in pixels")
+    valid_roi = Bool(required=True,
+                     description=("Boolean indicating if the ROI is a valid "
+                                  "cell or not"))
+    mask_matrix = List(List(Bool), required=True,
+                       description=("Bool nested list describing which pixels "
+                                    "in the ROI area are part of the cell"))
+    max_correction_up = Float(required=True,
+                              description=("Max correction in pixels in the "
+                                           "up direction"))
+    max_correction_down = Float(required=True,
+                                description=("Max correction in pixels in the "
+                                             "down direction"))
+    max_correction_left = Float(required=True,
+                                description=("Max correction in pixels in the "
+                                             "left direction"))
+    mask_image_plane = Int(required=True,
+                           description=("What tiff file the ROI lie "
```

Comment:
I would reword this to make it a bit more descriptive.

"The old segmentation pipeline stored overlapping ROIs on separate image planes. For compatibility purposes, this field must be kept, but will always be set to zero for the new updated pipeline"

Reply:
Updated